pkg/install/install_test.go

```diff
@@ -84,28 +74,7 @@ func TestDeployARMTemplate(t *testing.T) {
 					Wait(ctx, resourceGroup, deploymentName).
 					Return(wait.ErrWaitTimeout)
 			},
-			wantErr: wait.ErrWaitTimeout,
-		},
-		{
-			name: "Resource quota exceeded error",
-			mocks: func(dc *mock_features.MockDeploymentsClient) {
-				dc.EXPECT().
-					CreateOrUpdateAndWait(ctx, resourceGroup, deploymentName, deployment).
-					Return(quotaErr)
-			},
-			wantErr: cloudQuotaErr,
-		},
-		{
-			name: "Deployment active error, then resource quota exceeded error",
-			mocks: func(dc *mock_features.MockDeploymentsClient) {
-				dc.EXPECT().
-					CreateOrUpdateAndWait(ctx, resourceGroup, deploymentName, deployment).
-					Return(activeErr)
-				dc.EXPECT().
-					Wait(ctx, resourceGroup, deploymentName).
-					Return(quotaErr)
-			},
-			wantErr: cloudQuotaErr,
+			wantErr: "deployment of test ARM template failed",
```

Comment:
can you change api.CloudError.Error() so that it includes the details in a way that doesn't perturb all of our existing unit tests?  Then test the new details functionality with a test case?

Reply:
@jim-minter I've attempted to do this. Can you take another look?